gpu4pyscf/qmmm/pbc/itrf.py

```diff
@@ -0,0 +1,1131 @@
+# Author: Chenghan Li 
```

Comment:
Add the header similar to the other files. For example, https://github.com/pyscf/gpu4pyscf/blob/master/gpu4pyscf/pop/esp.py

Reply:
@MoleOrbitalHybridAnalyst The header is missing.